src/pages/product-page/ProductsPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect, useMemo } from "react"
+import { PRODUCTS_SEARCH } from "routes"
+import axios from "axios"
+import { useLocation } from "react-router"
+import "./product-page.scss"
+import GameCard from "components/game-card/GameCard"
+import { Game } from "types"
+
+const ProductsPage = () => {
+  const [searchGames, setSearchGames] = useState<Game[]>([])
+
+  const location = useLocation()
+  const search = new URLSearchParams(location.search)
+  let category = search.get("category")
```

Comment:
It is not crucial, but const could be used here. Shows that you'll never change that variable later

Reply:
Down to the file i re-assign the value of category. It's not very important but i thought it wood look slightly better

Reply:
Okay, haven't seen that. 

Could you create a helper called `capitalize`.

`const capitalize = (value:  string): string => {
  if (!value.length) return value;
  return `${value[0].toUpperCase()}${value.slice(1)}`
}`

Usage:

`const category = capitalize(search.get('category') || '')`

Try not to use variable mutations if possible.